addons/web_unsplash/static/src/components/media_dialog/image_selector.xml

```diff
@@ -24,16 +24,14 @@
         <a href="https://www.odoo.com/documentation/16.0/applications/websites/website/optimize/unsplash.html#generate-an-unsplash-application-id"
            class="mx-2" target="_blank">Application ID</a>
         here:
-        <div class="input-group d-flex justify-content-end align-items-center w-auto mx-2">
```

Comment:
This might break xpath, it's not stable compliant, can you find a way without modifying the DOM for 16.0 at least?